### DISCUSSION TITLE
how to interact with (or scroll in) the output of commands from helix command line

### DISCUSSION THREAD

**Discussion by goyalyashpal** [helix-editor/helix#5923] [Q&A] [answered]
so, i open `hx`, and run `:sh ls` - the huge list shows up with the scroll bar at its right side.

how can i scroll there?

![image](https://user-images.githubusercontent.com/19423063/218278039-fdef10fd-d8c3-4f6c-941d-e891d929815a.png)

**Comment by LEI** [comment, marked as answer]
It is possible to scroll with `<C-d>` (down) and `<C-u>` (up).
One could also use `:pipe ls` though it might require a `:new` buffer.

I wonder if there is a way to enter floating windows in order to move, yank, etc.
Neovim handles this nicely for [vim.lsp.buf.hover](https://neovim.io/doc/user/lsp.html#vim.lsp.buf.hover()) (equivalent to `<Space>k`):
> Calling the function twice will jump into the floating window

This would be nice for errors and diagnostics too (`<Space>e`?).
